pyiron_contrib/atomistics/meamfit/meamfit.py

```diff
@@ -0,0 +1,356 @@
+                                                    'Quantity to fit': [], 'Weights': []})
+        self._calculation_dataframe = self._calculation_dataframe.set_index('Job id')
+
+    @property
+    def calculation_dataframe(self):
+        return self._calculation_dataframe
+
+    @calculation_dataframe.setter
+    def calculation_dataframe(self, df):
+        self._calculation_dataframe = df
+
+    @property
+    def potential_performance_dataframe(self):
+        return self._potential_performance_dataframe
+
+    @property
+    def potential_timings_dataframe(self):
+        return self._potential_timings_dataframe
+
+    @property
+    def potentials(self):
+        return list(self._potential_timings_dataframe.index)
+
+    @property
+    def potential_paths(self):
+        return [posixpath.join(self.working_directory, pot) for pot in self.potentials]
+
+    @property
+    def structure(self):
+        """
+
+        Returns:
+
+        """
+        return None
+
+    @structure.setter
+    def structure(self, structure):
+        """
+
+        Args:
+            structure:
+
+        Returns:
+
+        """
+        raise ValueError('MeamFit has no structure!')
+
+    @property
+    def random_seed(self):
+        incar = self.input['SEED']
+        if incar == 'random':
+            self.input['SEED'] = random.randint(0, 100000)
+            incar = self.input['SEED']
+        return incar
+
+    @random_seed.setter
+    def random_seed(self, seed):
+        self.input['SEED'] = seed
+
+    def set_input_to_read_only(self):
+        """
+        This function enforces read-only mode for the input classes, but it has to be implement in the individual
+        classes.
+        """
+        self.input.read_only = True
+
+    def validate_ready_to_run(self):
+        pass
+
+    def write_input(self):
+        """
+        Call routines that generate the codespecifc input files
+
+        Returns:
+
+        """
+        if self.input['SEED'] == 'random':
+            self.input['SEED'] = random.randint(0, 100000)
+        self.input.write_file(file_name="settings", cwd=self.working_directory)
+        self._write_calc_db(calculation_dataframe=self._calculation_dataframe,
+                            file_name="fitdbse",
+                            cwd=self.working_directory)
+        self._copy_vasprun_xml(cwd=self.working_directory)
+
+    def add_job_to_fitting(self, job_id, time_step_start=0, time_step_end=-1, time_step_delta=10, quantity='E0',
+                           weight=[1.0, 0.0, 0.0]):
+        """
+
+        Args:
+            job_id (int):
+            time_step_start (int): initial timestep - after equilibration
+            time_step_end (int): last timestep to use
+            time_step_delta (int):
+            quantity (str): ['E', 'F'] for fitting the energies use 'E' and 'F' for fitting the forces
+            weight (list): default is one, but for lower temperatures use higher weights [100, 1000]
```

Comment:
Why is the default of length three?  What does it mean to use a weight >100-1000 if the default is 1?  A link to the official documentation would be enough.